rust/cadeau/src/xmf/vpx/mod.rs

```diff
@@ -77,7 +77,7 @@ pub struct VpxPacket<'a> {
 }
 
 impl VpxPacket<'_> {
-    /// # Safety
+    /// # SAFETY
```

Comment:
The section name should not be upper cased.

See how it’s done in the std library: https://doc.rust-lang.org/src/alloc/boxed.rs.html#1866-1880

Safety section in the documentation:
```rust
/// # Safety
///
/// `boxed_slice.len()` must be exactly `N`.
```

`SAFETY` comment above an unsafe block:
```rust
    // SAFETY: Pointer and allocator came from an existing box,
    // and our safety condition requires that the length is exactly `N`
    unsafe { Box::from_raw_in(ptr as *mut [T; N], alloc) }
```

This is to maximize tooling compatibility, let’s follow these conventions :slightly_smiling_face: